File: app/src/main/java/in/hridayan/ashell/fragments/setup/OnboardingItem2Fragment.java
package in.hridayan.ashell.fragments.setup;

import android.os.Bundle;
import android.view.Gravity;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.LinearLayout;
import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import in.hridayan.ashell.databinding.FragmentOnboardingItem2Binding;

public class OnboardingItem2Fragment extends Fragment {

  private FragmentOnboardingItem2Binding binding;

  @Override
  public View onCreateView(
      @NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
    binding = FragmentOnboardingItem2Binding.inflate(inflater, container, false);

    // We take 10% of the screen height as top margin an then assign a top margin to the disclaimer
    // title
    int screenHeight = getResources().getDisplayMetrics().heightPixels;
    int topMargin = (int) (screenHeight * 0.1);

    LinearLayout.LayoutParams params =
        new LinearLayout.LayoutParams(
            ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT);
    params.topMargin = topMargin;
    params.gravity = Gravity.CENTER_HORIZONTAL;

    binding.disclaimer.setLayoutParams(params);

    return binding.getRoot();
  }
}

File: app/src/main/java/in/hridayan/ashell/fragments/setup/OnboardingItem2Fragment.kt
package `in`.hridayan.ashell.fragments.setup

import android.os.Bundle
import android.view.Gravity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.LinearLayout
import androidx.fragment.app.Fragment
import `in`.hridayan.ashell.databinding.FragmentOnboardingItem2Binding

class OnboardingItem2Fragment : Fragment() {
    private lateinit var binding: FragmentOnboardingItem2Binding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?
    ): View {
        binding = FragmentOnboardingItem2Binding.inflate(inflater, container, false)

        // We take 10% of the screen height as top margin an then assign a top margin to the disclaimer
        // title
        val screenHeight = resources.displayMetrics.heightPixels
        val topMargin = (screenHeight * 0.1).toInt()

        val params =
            LinearLayout.LayoutParams(
                ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT
            )
        params.topMargin = topMargin
        params.gravity = Gravity.CENTER_HORIZONTAL

        binding.disclaimer.layoutParams = params

        return binding.root
    }
}
